README.md

```diff
@@ -107,8 +106,8 @@ The precise steps described do not need to be followed exactly, but the results
 
 To test a specific version:
 
-* For 2019-09 and later published drafts, implementations that are able to detect the draft of each schema via `$schema` SHOULD be configured to do so
-* For draft-07 and earlier, draft-next, and implementations unable to detect via `$schema`, implementations MUST be configured to expect the draft matching the test directory name
+* For 2019-09 and later published versions, implementations that are able to detect the version of each schema via `$schema` SHOULD be configured to do so
+* For draft-07 and earlier, v1 (not yet released), and implementations unable to detect via `$schema`, implementations MUST be configured to expect the version matching the test directory name
```

Comment:
I'm not sure why draft-next was included in this bullet. That doesn't seem correct to me and I don't think it's correct for v1 either.


```suggestion
* For draft-07 and earlier, and implementations unable to detect via `$schema`, implementations MUST be configured to expect the version matching the test directory name
```

Reply:
I'm going to leave this for now, as it's still not clear to me (nor in the latest version of the spec documents in json-schema-spec main branch) whether the name of the draft is actually intended to be "v1" or "v1/<year>" and how `$schema` keywords would refer to it.

As far as the test suite is concerned, as long as everything consistently refers to "v1" then test harnesses or implementations can just skip this directory entirely for now, as they would have done for "draft-next".